src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form>
+      <h1> FORM </h1>
+      <p> Будь ласка заповність форму</p>
+      <script type="text/javascript" src="./main.js"></script>
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+     Surname:
+          <input type="text" name="username"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+        Name:
+          <input type="text" name="username"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+        How old are You?
+          <input type="ages" name="ages"
+            minlength="1"
+            maxlength="100"
+            pattern="\d+"
+            required
+           >
```

Comment:
the attributes should be placed from a new line each
```suggestion
          <input
            type="ages"
            name="ages"
            minlength="1"
            maxlength="100"
            pattern="\d+"
            required
           >
```